src/Sulu/Bundle/AdminBundle/Resources/js/components/Table/Cell.js

```diff
@@ -46,7 +46,7 @@ export default class Cell extends React.PureComponent<Props> {
                 className={cellClass}
                 colSpan={colSpan}
             >
-                <div className={tableStyles.cellContent} style={style}>
+                <div className={tableStyles.cellContent} style={style} title={children[1]}>
```

Comment:
as this is the general `Cell` component of the table, the `children` property coul includes things like a button or a select. i think the better place to do this would be the `StringFieldTransformer`. 

the `StringFieldTransformer` is called with the string value and returns a html node for it. i would apply the styling to the returned node like the `ColorFieldTransformer.js` already does 🙂